### DISCUSSION TITLE
Accepted Proofs of Academic Status

### DISCUSSION THREAD

**Discussion by Caelan27** [community/community#141335] [GitHub Education] [answered]
### Hi everyone,

I am a secondary school student in the UK trying to join Github Education. However, I can't think of what to use to verify my student status. I don't have a student ID. According to https://github.com/orgs/community/discussions/111352, the accepted proof types are:
1. Student ID
2. Transcript
3. Enrollment or affiliation letter
4. Class schedule

The only thing I have is a class schedule, but it doesn't say the name of my school on it. Are there any other ways I can verify?

I just had the idea of using a report card, since I'll be getting one in around a week. I think it should have all the required information on it but I'm not sure whether I need a more formal document to prove I'm a student.

Thanks!

**Comment by marlonwq** [comment]
> ### Hi everyone,
> I am a secondary school student in the UK trying to join Github Education. However, I can't think of what to use to verify my student status. I don't have a student ID. According to https://github.com/orgs/community/discussions/111352, the accepted proof types are:
> 
> 1. Student ID
> 2. Transcript
> 3. Enrollment or affiliation letter
> 4. Class schedule
> 
> The only thing I have is a class schedule, but it doesn't say the name of my school on it. Are there any other ways I can verify?
> 
> I just had the idea of using a report card, since I'll be getting one in around a week. I think it should have all the required information on it but I'm not sure whether I need a more formal document to prove I'm a student.
> 
> Thanks!

Hello, could you insert the document you received when you registered in school.

**Reply by Caelan27** [reply]
I'm not sure if I have a document like that. I'll ask my parents if they have it but unless it was given directly to them, I don't remember ever receiving one.

**Reply by marlonwq** [reply]
Ask them, I redeemed mine with a document like this, if you don't have it maybe the bulletin will work

**Reply by marlonwq** [reply]
> I'm not sure if I have a document like that. I'll ask my parents if they have it but unless it was given directly to them, I don't remember ever receiving one.

It achieved?

**Reply by Caelan27** [reply]
Not yet but I will ask my school about it on Monday. My parents do not have any documents from when I registered.

**Comment by ranjith-fullstack** [comment]
can u tell once again

**Comment by t1noo7** [comment]
Hi @Caelan27 ,

Once I read your thread, you have only a class schedule, and you're finding the student card by asking your parents, but not sure. Here's the requirement from the doc by the link you shared:

> To be eligible for GitHub Education, including the GitHub Student Developer Pack and other benefits, you must:
> 
> Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution
> Have a verifiable school-issued email address or upload documents that prove your current student status
> Have a [GitHub personal account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github)
> Be at least 13 years old

Looks like you're fine but one important thing that I am scared about your situation, that's an email school-issued. It is the first and foremost thing when you start entering your registration information. You can try creating a [ticket](https://support.github.com/contact/education), choose `I am or would like to be a Partner for the ...`, and tell the staff your issue: what resources you have, what you can do, what you are limited to, do so on and wait for their reply for the next action.

Thinking you can mention sending them a letter of admission to the school you are studying (if you still have it) along with an identification card for comparison if more authenticity is needed in the content of the ticket.

**Reply by Caelan27** [reply]
I have a school email address and it automatically detected my school based on the domain, so I'm pretty sure it's valid. My parents do not have any documents from when I registered to the school. I will ask my school if they can give me some kind of verification that I am a student on Monday.

**Comment by Caelan27** [comment, marked as answer]
I was approved! I just used the report card I got today and it  worked! I thought a person would have had to manually approve me but I don't think they did, since it happen in less than a minute.
